multimodal/src/autogluon/multimodal/data/process_image.py

```diff
@@ -395,14 +398,21 @@ def process_one_sample(
         """
         images = []
         zero_images = []
+        mm_data = dict(img_prefix=None, bbox_fields=[])
         ret = {}
         column_start = 0
-        for per_col_name, per_col_image_paths in image_paths.items():
-            for img_path in per_col_image_paths[: self.max_img_num_per_col]:
-                if self.prefix == MMDET_IMAGE or self.prefix.lower().startswith(MMOCR):
-                    data = dict(img_info=dict(filename=img_path), img_prefix=None)
-                    images.append(self.val_processor(data))
+        if self.prefix == MMDET_IMAGE or self.prefix.lower().startswith(MMOCR):
```

Comment:
Consider simplifying the condition: ` if self.prefix.lower().startswith(MMCV_MODELS):`.

Reply:
they both start with mm, this works for now but may cause conflict in the future? or shall we rename MMDET_IMAGE and MMOCR to start with same prefix like "mmlab_detection", "mmlab_ocr"

Reply:
I like "mmlab_detection", "mmlab_ocr", we can discuss later.

Reply:
Changed the constant MMCV_MODELS to MMLAB_MODELS and updated the logic here